core/src/HIP/Kokkos_HIP_Abort.hpp

```diff
@@ -50,17 +50,7 @@
 
 #include <hip/hip_runtime.h>
 
-// FIXME_HIP ROCm 4.5 version header include would be <rocm/rocm_version.h>
-#if __has_include(<rocm_version.h>)
 #include <rocm_version.h>
```

Comment:
```suggestion
```
What's the reason for keeping this include file?

Reply:
I don't think you get the version if you don't include

Reply:
Right, but we are not using the version anywhere anymore AFAICT. I don't care much either way but just want this to be a conscious choice.

Reply:
You are right, this is not needed anymore. I've removed it.